lib/Conversion/TorchToTcp/Elementwise.cpp

```diff
@@ -116,7 +256,20 @@ void torch_to_tcp::populateElementwisePatternsAndLegality(
 
   target.addIllegalOp<AtenTanhOp>();
   patterns.add<ConvertAtenTanhOp>(typeConverter, context);
+  target.addIllegalOp<AtenClampOp>();
+  patterns.add<ConvertAtenClampOp>(typeConverter, context);
+  target.addIllegalOp<AtenReluOp>();
+  target.addIllegalOp<AtenRelu6Op>();
+  patterns.add<ConvertAtenReluOp<AtenReluOp>>(typeConverter, context);
+  patterns.add<ConvertAtenReluOp<AtenRelu6Op>>(typeConverter, context);
```

Comment:
Is there a reason the e2e test for `Relu6` is not passing? Every time support for a new op is added, it should be accompanied by at least one passing e2e test. See: https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#dos-and-donts-for-unit-vs-end-to-end-testing

Reply:
Thats a good question. It looks like the `relu6` e2e test ends up doing a `relu` followed by a `min` (which is not supported in Tcp yet). Here is the module I see before `TorchToTcp` conversion:
```
  func.func @forward(%arg0: !torch.vtensor<[?,?],f32>) -> !torch.vtensor<[?,?],f32> {
    %int6 = torch.constant.int 6
    %false = torch.constant.bool false
    %none = torch.constant.none
    %0 = torch.aten.relu %arg0 : !torch.vtensor<[?,?],f32> -> !torch.vtensor<[?,?],f32>
    %1 = torch.prim.ListConstruct  : () -> !torch.list<int>
    %2 = torch.prim.NumToTensor.Scalar %int6 : !torch.int -> !torch.vtensor<[],si64>
    %3 = torch.aten.to.dtype %2, %int6, %false, %false, %none : !torch.vtensor<[],si64>, !torch.int, !torch.bool, !torch.bool, !torch.none -> !torch.vtensor<[],f32>
    %4 = torch.aten.broadcast_to %3, %1 : !torch.vtensor<[],f32>, !torch.list<int> -> !torch.vtensor<[],f32>
    %5 = torch.aten.minimum %0, %4 : !torch.vtensor<[?,?],f32>, !torch.vtensor<[],f32> -> !torch.vtensor<[?,?],f32>
    return %5 : !torch.vtensor<[?,?],f32>
  }
```

Is this a known issue?

Reply:
`relu6` can be represented as `min(relu(x), 6)` so the module above is sort of expected. Perhaps we can hold off on `relu6` until we have `tcp.min`, or capture a TODO to re-enable the e2e test once `tcp.min` is added.

Reply:
@sjain-stanford While that representation is correct, the fact that it is represented that way when torch has a `torch.aten.relu6` op seems like a bug to me. 

Both those options seem reasonable to me.

Reply:
Ah I see. In that case I agree that I'm not sure why it would lower to these primitives, or if it is somehow configurable to disable this.

Reply:
Ah right, this is because we have a decomposition for `Relu6`: https://github.com/llvm/torch-mlir/blob/65846010cbfd121c8666ec3a44827f975600e081/lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp#L745

You should be able to turn off the decomposition by adding your ops to your backend here:

https://github.com/llvm/torch-mlir/blob/65846010cbfd121c8666ec3a44827f975600e081/python/torch_mlir/__init__.py#L250

Reply:
Or if it's not urgent to have support for `Relu6`, you can just remove the `Relu6` pattern and wait until support for `min` is added. Either approach is fine

Reply:
Ah okay. Good to know that is configurable. Thanks for the pointer.

For now, I've removed support for `Relu6`.